doc/basics.md

```diff
@@ -151,7 +180,7 @@ behavior.
 
 ### Error Handling Example
 
-This is how the example in "Using the Parsed JSON" could be written using only error code checking:
+This is how the example in "Using the Parsed JSON" could be written using only error code checking):
```

Comment:
You appear to have added a closing parenthesis for no good reason that I can spot.